debian/stocks-server.postinst

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+set -e
+set -x
```

Comment:
Are all the `set -x` Debian best practice or only debugging leftovers?

Reply:
Neither. This isn't really a Debian package, and I'd like to be explicit about what is happening behind the scenes for the user. I can also remove these though.

Reply:
They may stay :)